libsolidity/analysis/TypeChecker.h

```diff
@@ -166,6 +166,9 @@ class TypeChecker: private ASTConstVisitor
 	/// Flag indicating whether we are currently inside a StructDefinition.
 	bool m_insideStruct = false;
 
+	/// Flag indicating whether we are currently inside the invocation of a modifier
+	bool m_insideModifierInvocation = false;
```

Comment:
Should this be `m_insideFunctionSignature` instead because if I understood correctly what we care about is whether we are evaluating the function signature or not?

Reply:
Hmm I suppose, though I do explicitly set it only for the processing of the identifier part of the modifier and not the signature..